src/main/java/uk/gov/ons/census/notifyprocessor/service/EnrichedFulfilmentRequestService.java

```diff
@@ -28,13 +39,36 @@ public void processMessage(EnrichedFulfilmentRequest fulfilmentRequest) {
           Map.of("uac", fulfilmentRequest.getUac()),
           UUID.randomUUID().toString(),
           senderId);
-    } catch (NotificationClientException e) {
+    } catch (NotificationClientException notificationClientException) {
+      handleNotificationClientException(notificationClientException);
+    }
+  }
+
+  private void handleNotificationClientException(
+      NotificationClientException notificationClientException) {
+    String exceptionMessage = notificationClientException.getMessage();
+
+    if (exceptionMessage.contains(BAD_REQUEST_ERROR)
+        && exceptionMessage.contains(CANNOT_SEND_INTERNATIONAL_MESSAGE)) {
+      log.with("problem", CANNOT_SEND_INTERNATIONAL_MESSAGE).warn(PHONE_NUMBER_FAILED_VALIDATION);
```

Comment:
Could we log the transaction ID or case ID? We'd need something to be able to manually track down what request/case failed.